src/iinit.cpp

```diff
@@ -359,7 +363,7 @@ int main( int argc, char **argv )
             }
 
             printErrorStack(Conn->rError);
-            if (ttl > 0 && irods::AUTH_PAM_SCHEME != lower_scheme) {
+            if (ttl > 0 && lower_scheme != irods_auth::scheme::pam_password) {
```

Comment:
The plugin with scheme AUTH_PAM_SCHEME is the legacy plugin. This is in the `else` case for `if (use_legacy_authentication)`, so we only deal with the new plugins in this part.

Unfortunately, we kind of have to do it this way because of how authentication works while we try to bridge the gap between the old and the new.

Reply:
I suppose it wouldn't hurt to check for that scheme, it's just that we aren't going to be shipping a plugin with that scheme besides the legacy plugin. We could actually probably just perform the check once at the top and set a flag if that seems better.

Reply:
Good enough for me. I was just curious. The code is fine as is.

Reply:
Okey dokey. I'll just leave this, then. Thanks